testsuite/features/secondary/min_centos_salt.feature

```diff
@@ -65,12 +65,22 @@ Feature: Be able to bootstrap a CentOS minion and do some basic operations on it
     And I wait until event "Subscribe channels scheduled by admin" is completed
 
 @centos_minion
-  Scenario: Detect latest Salt changes on the CentOS minion
-    When I query latest Salt changes on "ceos_minion"
+  Scenario: Prepare the SSH-managed CentOS minion
+    Given I am authorized
+    When I enable SUSE Manager tools repositories on "ceos_ssh_minion"
+    And  I enable repository "CentOS-Base" on this "ceos_ssh_minion"
+    And  I install package "hwdata m2crypto wget" on this "ceos_ssh_minion"
+    And  I install package "spacewalk-client-tools spacewalk-check spacewalk-client-setup mgr-daemon mgr-osad mgr-cfg-actions" on this "ceos_ssh_minion"
```

Comment:
For traditional clients sure, but for SSH minion I'm not sure that we need to install those packages.